src/org/labkey/test/tests/DomainDesignerTest.java

```diff
@@ -824,12 +845,28 @@ public void setFieldAsVariable() throws Exception
         domainDesignerPage.clickSaveAndFinish();
 
         DomainResponse domainResponse = dgen.getDomain(createDefaultConnection(true));
+        Map<String, Object> testColumnProperties = getPropertyPerColumn(domainResponse.getColumns(), "recommendedVariable",
+                "variableField", "extraField");
+        assertEquals("variableField should have recommendedVariable marked true", true, testColumnProperties.get("variableField"));
+        assertEquals("extraField should not have recommendedVariable marked true", false, testColumnProperties.get("extraField"));
+
         Map<String, Object> variableCol = domainResponse.getColumns().stream()
                 .filter(a->a.get("name").equals("variableField")).findFirst().orElse(null);
         Map<String, Object> extraFieldCol = domainResponse.getColumns().stream()
                 .filter(a->a.get("name").equals("extraField")).findFirst().orElse(null);
-        assertEquals("expect column to have measure enabled", true, variableCol.get("recommendedVariable"));
-        assertEquals("expect column not to have measure enabled", false, extraFieldCol.get("recommendedVariable"));
+        assertEquals("expect column to have recommendedVariable enabled", true, variableCol.get("recommendedVariable"));
+        assertEquals("expect column not to have recommendedVariable enabled", false, extraFieldCol.get("recommendedVariable"));
```

Comment:
Remove old, duplicate asserts. They have been superseded by the asserts above.

Reply:
Yes, they check more or less the same thing.  But having them there costs very little and provides a little bit of data for whoever needs to investigate state later if there are failures